### ISSUE TITLE
stat_seg_length.ipynb: ошибка при выполнении plot_seg_len_stats

### ISSUE THREAD

**Issue by Borpa** [fizmat/hopfield-tracking#18]
![изображение](https://user-images.githubusercontent.com/114162087/191788589-8bc61d16-8e05-4f2a-93ef-380f68dbed68.png)

**Comment by fizmat** [comment]
Да, stat_seg_length требует полный датасет bman. Надо это указать в блокноте и желательно в readme.

**Comment by Alyaski** [comment]
взял в работу.
